yarn.lock

```diff
@@ -6707,6 +6707,11 @@ slice-ansi@^4.0.0:
     astral-regex "^2.0.0"
     is-fullwidth-code-point "^3.0.0"
 
+slick@^1.12.2:
```

Comment:
Don't commit yarn lock file, it's generated by webpack build

Reply:
Solved